bgl_babylon.cc

```diff
@@ -94,9 +94,6 @@ bool Babylon::open()
     return false;
   }
 
-  if( fseek( f, i, SEEK_SET ) ) /* can't seek - emulate */
-      for(int j=0;j < i - 6;j++) fgetc( f );
```

Comment:
Is this diff related to QtWebEngine somehow?

Reply:
Why do you keep such a long commit history if even you, the author, can't use it to figure out why something has been changed?